tests/test_opencovercoverage.php

```diff
@@ -0,0 +1,78 @@
+<?php
+//
+// After including cdash_test_case.php, subsequent require_once calls are
+// relative to the top of the CDash source tree
+//
+require_once(dirname(__FILE__).'/cdash_test_case.php');
+
+class JSCoverCoverageTestCase extends KWWebTestCase
+{
+  function __construct()
+    {
```

Comment:
The code style doesn't conform to the rest of CDash, not sure why the style checker isn't catching it.